integration/src/main.rs

```diff
@@ -127,6 +128,16 @@ impl TestArgs {
         Ok(erc20)
     }
 
+    /// Fund an address with the given erc20
```

Comment:
nit: this doc comment implies that we can select the address that gets funded when really it's just `self.wallet_addr`, can we tweak this?